docs/marketplace/dev/front-end/extending-the-project/index.md

```diff
@@ -22,7 +22,7 @@ Prior to starting the project extension, verify that the marketplace modules are
 | SalesMerchantPortalGui (optional)           | >= 0.8.1  |
 | SecurityMerchantPortalGui (optional)        | >= 0.4.2  |
 
-If not, follow the steps from the [Migration guide: Extending the project](/docs/marketplace/dev/front-end/extending-the-project/migration-guide-extending-the-project.html).
+Ensure that you have followed the [migration guide for extending the project](/docs/marketplace/dev/front-end/extending-the-project/migration-guide-extending-the-project.html) prior to extending the project.
```

Comment:
```suggestion
Before extending the project, ensure that you have followed the [migration guide for extending the project](/docs/marketplace/dev/front-end/extending-the-project/migration-guide-extending-the-project.html).
```
The sequence of phrases in sentences should correspond to the order you want a user do smth. In this case, they should first write what should be done first, i.e. first is "before extending the project", and then "ensure..."
Also, "prior" is already used in previous sentence, so I suggest "before" in this one.